apps/dashr-ncbi-explorer/init.R

```diff
@@ -0,0 +1,21 @@
+# R script to run author supplied code, typically used to install additional R packages
+# contains placeholders which are inserted by the compile script
+# NOTE: this script is executed in the chroot context; check paths!
+
+r <- getOption("repos")
+r["CRAN"] <- "http://cloud.r-project.org"
+options(repos=r)
+
+
+
+# ======================================================================
+# packages go here
+remotes::install_github("plotly/dashR", dependencies=TRUE)
```

Comment:
@HammadTheOne I think you want to use `upgrade = TRUE` here as well.

Reply:
Done! Thanks for pointing it out.